application/controllers/admin.php

```diff
@@ -0,0 +1,346 @@
+                    'display_column' => 'school'
+                ),
+                'accommodation' => array(
+                    'type' => 'boolean'
+                ),
+                'meal' => array(
+                    'type' => 'boolean'
+                ),
+                'race' => array(
+                    'type' => 'enum',
+                    'enum' => $GLOBALS['RACES']
+                )//,
+                //'team_id' => array(
+                    //'type' => 'foreignkey',
+                    //'references' => 'team'
+                //)
+            ),
+            /*'team' => array(
+                'first' => array(
+                    'type' => 'foreignkey',
+                    'references' => 'people',
+                    'display_column' => 'name'
+                ),
+                'second' => array(
+                    'type' => 'foreignkey',
+                    'references' => 'people',
+                    'display_column' => 'name'
+                ),
+                'third' => array(
+                    'type' => 'foreignkey',
+                    'references' => 'people',
+                    'display_column' => 'name'
+                ),
+                'school_id' => array(
+                    'type' => 'foreignkey',
+                    'references' => 'users',
+                    'display_column' => 'name'
+                )
+            ),
+             */
+            'user' => array(
+                'school' => array(
+                    'type' => 'varchar',
+                    'len' => 30
+                ),
+                'leader' => array(
+                    'type' => 'varchar',
+                    'len' => 10
+                ),
+                'tel' => array(
+                    'type' => 'varchar',
+                    'len' => 11
+                ),
+                'mail' => array(
+                    'type' => 'varchar',
+                    'len' => 30
+                ),
+                'password' => array(
+                    'type' => 'password'
+                ),
+                'bill' => array(
+                    'type' => 'int',
+                    'len' => 11
+                ),
+                'paid' => array(
+                    'type' => 'boolean'
+                ),
+                'confirmed' => array(
+                    'type' => 'boolean'
+                ),
+                'association_name' => array(
+                    'type' => 'varchar',
+                    'len' => 15
+                ),
+                'province' => array(
+                    'type' => 'enum',
+                    'enum' => $GLOBALS['PROVINCES']
+                ),
+                'group_id' => array(
+                    'type' => 'foreignkey',
+                    'references' => 'group',
+                    'display_column' => 'group_name'
+                )
+            )
+        );
+    }
+
+    private function get_captcha() {
```

Comment:
I wonder whether we can make this as a GLOBAL helper as we may need it in other place.